docs/labwc-config.5.scd

```diff
@@ -169,6 +169,7 @@ The rest of this man page describes configuration options.
 	modifiers include S (shift); C (control); A (alt); W (super). Unlike
 	Openbox, multiple space-separated key combinations and key-chains are
 	not supported.
+	Use wev (wayland event viewer) to view all available keynames.
```

Comment:
Suggest moving the start of the sentence to the previous line, potentially breaking to the next one.
E.g. something like this:

	Define a key binding in the format *modifier-key*, where supported
	modifiers include S (shift); C (control); A (alt); W (super). Unlike
	Openbox, multiple space-separated key combinations and key-chains are
	not supported. The application "wev" (wayland event viewer) can be
	used to view all available keynames.

Reply:
I have moved it and added a hint to look for wev in the distribution's repositories.